src/kits/arm/arm_node.cpp

```diff
@@ -22,7 +22,26 @@ namespace ros {
 
 class ArmNode {
 public:
-  ArmNode(arm::Arm& arm, const Eigen::VectorXd& home_position) : arm_(arm), home_position_(home_position) { }
+  ArmNode(arm::Arm& arm, const Eigen::VectorXd& home_position, const std::vector<double>& ik_seed) : arm_(arm), home_position_(home_position) {
+    if (ik_seed.size() > 0) {
```

Comment:
Small thing, but my preference would be checking `home_position.size() == ik_seed.size()` here, so we never have an invalid IK seed.

Reply:
I updated the way setting the IK seed works, so you don't have to have the seed ready when initializing the node/can change it in code later. Added this check into the new setter function though.